src/content/en/2020/third-parties.md

```diff
@@ -164,89 +189,109 @@ On the other end of the spectrum, the categories CDN, Content and Hosting all re
   width=600,
-  height=371
+  height=371,
+  sheets_gid="1363055589",
+  sql_file="percent_of_third_party_cache.sql"
   )
 }}
 
 Breaking down by response type highlights some common offenders: xml and text responses are less likely to be cacheable. Surprisingly, less than two-thirds of images served by third-parties are cacheable. On further inspection, this is due to the use of tracking 'pixels' which are returned as non-cacheable zero-size image responses.
 
+{# TODO authors - I believe this could be explained more. Either here or in the paragraph above the chart. Tracking pixels are one of the few third-party requests that should be non-cacheable but this is neither mentioned above, nor really explained here as working as intended. #}
+
 ### Large redirects
 
 Many third-parties result in redirect responses, i.e. HTTP status codes 3XX. These occur due to the use of vanity domains or to share information across domains through request headers. This is especially true for advertising networks. Large redirect responses are an indication of a misconfiguration, as the response should be around 340B for a valid `Location` response header plus overheads. The chart below shows the distribution of body size for all third-party redirects in the HTTP Archive.
 
 {{ figure_markup(
   image="redirects-body-size.png",
-  caption="Most redirects are under 420B, the 99th percentile is over 30kB!",
-  description="Distribution of redirect body sizes showing 90% are under 420B, 1% are over 30kB and 0.1% are over 100kB",
+  caption="Distribution of third-party 3XX body size",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vSNXusoFJKi8Z1i_yuG_5umUCJOZwBMaZaECq4T8lU5zIQuLR7UHfZOJmXsXzPWQTJFnFa3dcOEPJgy/pubchart?oid=1145900631&format=interactive",
+  description="Distribution of redirect body sizes showing 90% are under 420 B, 1% are over 30 kB and 0.1% are over 100 kB.",
   width=600,
-  height=371
+  height=371,
+  sheets_gid="1056232541",
+  sql_file="distribution_of_3XX_response_body_size.sql"
   )
 }}
 
-The results show that the majority of 3XX responses are small: the 90th percentile is 420B, i.e. 90% of 3XX responses are 420 bytes or smaller. The 95th percentile is 6.5kB, the 99th is 36kB and the 99.9th is over 100kB! Whilst redirects may seem innocuous, 100kB is an unreasonable amount of bytes over the wire for a response that simply leads to another response.
+The results show that the majority of 3XX responses are small: the 90th percentile is 420 bytes, i.e. 90% of 3XX responses are 420 bytes or smaller. The 95th percentile is 6.5 kB, the 99th is 36 kB and the 99.9th is over 100 kB! Whilst redirects may seem innocuous, 100kB is an unreasonable amount of bytes over the wire for a response that simply leads to another response.
+
+{# TODO Authors/Analysts - is it weird to use kB (1000) instead of KB (1024)? #}
```

Comment:
@simonhearne WDYT?

Reply:
Looking through the almanac it seems more common to use Kilobyte (kB) than Kibibyte (KiB). The kilobyte prefix is a lower case k according to ISO 80000:13

Reply:
OK happy to leave, so will resolve this. At least it's clear what this is as opposed to the ambiguous KB!